src/executors/architect.yaml

```
@@ -1,3 +1,3 @@
 docker:
   - entrypoint: /bin/bash
-    image: quay.io/giantswarm/architect:3.4.0
+    image: quay.io/giantswarm/architect:3.4.0-4ab0dc4e463de48b0913937e63cda54919045a71
```

Apply once architect [3.4.1](https://github.com/giantswarm/architect/releases/tag/v3.4.1) has been released

```suggestion
    image: quay.io/giantswarm/architect:3.4.1
```